docker/production/Dockerfile.frontend

```
@@ -23,8 +23,9 @@ COPY apps ./apps
 RUN pnpm install --frozen-lockfile
 RUN pnpm --filter frontend build
 
-FROM nginx:1.27-alpine AS runner
+FROM georgjung/nginx-brotli AS runner
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)